src/NUnitFramework/tests/Attributes/RangeAttributeTests.cs

```diff
@@ -414,21 +463,58 @@ private void CheckValues(string methodName, params object[] expected)
             var method = GetType().GetMethod(methodName, BindingFlags.NonPublic | BindingFlags.Instance);
             var param = method.GetParameters()[0];
 #if NETSTANDARD1_3 || NETSTANDARD1_6
-            var attr = param.GetCustomAttributes(typeof(ValuesAttribute), false).First() as ValuesAttribute;
+            var attr = param.GetCustomAttributes(typeof(RangeAttribute), false).First() as RangeAttribute;
 #else
-            var attr = param.GetCustomAttributes(typeof(ValuesAttribute), false)[0] as ValuesAttribute;
+            var attr = param.GetCustomAttributes(typeof(RangeAttribute), false)[0] as RangeAttribute;
 #endif
             Assert.That(attr.GetData(new ParameterWrapper(new MethodWrapper(GetType(), method), param)), Is.EqualTo(expected));
         }
 
+        private void CheckValuesWithMultipleAttributes(string methodName, params object[] expected)
```

Comment:
I'm not sure this is totally the right test - I think what you're testing here would be the same as testing two range attributes in two separate tests? Unless I'm missing something extra?

The think I was thinking more is that the test building logic could resolve two attributes on the same parameter. Take a look at `NUnit.Framework.Attributes.TheoryTests.AllValuesMayBeSuppliedByAttributes`  - and how that uses `TestBuilder.MakeParameterizedMethodSuite()` and then checks the number of tests generated. I think we would need a test like that, if we wanted to make sure there was no regression here. 🙂

Reply:
Ah, got it. I updated the tests to use `TestBuilder.MakeParameterizedMethodSuite`.